layouts/html.hbs

```diff
@@ -85,7 +85,8 @@
       });
       {{/babel}}
     </script>
-    <script src="{{relativePath}}/bundle.js" data-webpack-inline></script>
+    <script src="{{relativePath}}/bundle.js" type="module" data-webpack-inline></script>
+    <script src="{{relativePath}}/bundle-legacy.js" nomodule></script>
```

Comment:
I think we might need to chat with Product about this change. It has some significant upgrade implications. If somebody has forked `html.hbs`, their site will stop working in IE11 if they're not careful during the upgrade. The `bundle.js` asset no longer points to the same thing post-upgrade.

Reply:
talked to Rose and she gave the ok! people who overrode html.hbs will have to refresh their html.hbs anyways for the dev sdk branch change